### PR TITLE
Client first version

### PR DESCRIPTION
**Pull request by woolfg**

### CODE REVIEW

File: webapp/pages/anchor.vue [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<template>
+  <div>
+    <h1>Anbindung der Anchor-Analytics Daten an OpenPodcast</h1>
+    <p class="my-4">
+      Um Anchor an OpenPodcast anzubinden, musst du dich bei Anchor anmelden.<br /><br />
```
Comment (mre) [inline webapp/pages/anchor.vue:5]:
```suggestion
      Um Anchor an OpenPodcast anzubinden, musst Du dich bei Anchor anmelden.<br /><br />
```

Just an idea.

File: webapp/pages/anchor.vue [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            />
+            <path
+              d="M93.9676 39.0409C96.393 38.4038 97.8624 35.9116 97.0079 33.5539C95.2932 28.8227 92.871 24.3692 89.8167 20.348C85.8452 15.1192 80.8826 10.7238 75.2124 7.41289C69.5422 4.10194 63.2754 1.94025 56.7698 1.05124C51.7666 0.367541 46.6976 0.446843 41.7345 1.27873C39.2613 1.69328 37.813 4.19778 38.4501 6.62326C39.0873 9.04874 41.5694 10.4717 44.0505 10.1071C47.8511 9.54855 51.7191 9.52689 55.5402 10.0491C60.8642 10.7766 65.9928 12.5457 70.6331 15.2552C75.2735 17.9648 79.3347 21.5619 82.5849 25.841C84.9175 28.9121 86.7997 32.2913 88.1811 35.8758C89.083 38.2158 91.5421 39.6781 93.9676 39.0409Z"
+              fill="currentFill"
+            />
+          </svg>
+        </div>
+      </div>
+      <div v-if="showForm">
+        <div v-if="errorMsg" class="p-2 bg-red-100 rounded-md flex">
```
Comment (mre) [inline webapp/pages/anchor.vue:33]: What do you think about centering the input form within the box?

File: webapp/pages/anchor.vue [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    // Simulate loading time
+    setTimeout(() => {
+      this.showSpinner = false;
+      this.showForm = true;
+    }, 1);
+  },
+  methods: {
+    login: async function () {
+      if (!this.email || !this.password) {
+        this.errorMsg = "Bitte gib deine Zugangsdaten vollständig ein.";
```
Comment (mre) [inline webapp/pages/anchor.vue:98]:
```suggestion
        this.errorMsg = "Bitte gib Deine Zugangsdaten vollständig ein.";
```

File: webapp/pages/anchor.vue [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      this.showForm = false;
+      this.showSpinner = true;
+      // send cors request to api
+      const response = await useFetch(this.baseURL + "/connect/anchor", {
+        method: "POST",
+        methods: "cors",
+        body: JSON.stringify({ email: this.email, password: this.password }),
+      });
+      console.log(response);
+      // if response is bnot 200 show form again and display error message
```
Comment (mre) [inline webapp/pages/anchor.vue:112]:
```suggestion
      // if response is not, 200 show form again and display error message
```

File: webapp/pages/anchor.vue [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+      if (response.status !== 200) {
+        this.showForm = true;
+        this.showSpinner = false;
+        this.errorMsg =
+          "Dein Podcast konnte nicht angebunden werden, bitte überprüfe die angegebenen Zugangsdaten und versuche es erneut.";
```
Comment (mre) [inline webapp/pages/anchor.vue:117]:
```suggestion
          "Dein Podcast konnte nicht angebunden werden. Bitte überprüfe die angegebenen Zugangsdaten und versuche es erneut.";
```

File: webapp/pages/anchor.vue [hunk 6]
```diff
@@ -0,0 +1,131 @@
+<template>
+  <div>
+    <h1>Anbindung der Anchor-Analytics Daten an OpenPodcast</h1>
+    <p class="my-4">
+      Um Anchor an OpenPodcast anzubinden, musst du dich bei Anchor anmelden.<br /><br />
+      Die Zugangsdaten werden nicht gespeichert und dienen nur zur einmaligen
+      Anbindung. Die Anbindung muss in regelmäßigen Abständen erneuert werden,
+      wir informieren dich aber rechtzeitig.
```
Comment (mre) [inline webapp/pages/anchor.vue:8]:
```suggestion
      wir informieren Dich aber rechtzeitig.
```

File: webapp/README.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+Look at the [Nuxt 3 documentation](https://nuxt.com/docs/getting-started/introduction) to learn more.
+
+## Setup
+
+Make sure to install the dependencies:
+
+```bash
+# yarn
+yarn install
```
[the remark this answers is not in the document]
Comment (woolfg) [reply webapp/README.md:11]: that is just the default readme

**Pull request merged**